static/js/compose.js

```diff
@@ -658,6 +662,9 @@ export function initialize() {
         e.preventDefault();
         e.stopPropagation();
 
+        // Hide unneeded compose_control_buttons in preview mode.
+        $("#compose").addClass("preview_mode");
```

Comment:
I think we can just do `$(".preview_mode_hidden").show()` ?

Reply:
Okay, I will try this and let you know.

Reply:
FYI #20993 is also ready.

Reply:
if we use hide/show it is having little CSS problem.
![hiding_compose_buttons](https://user-images.githubusercontent.com/41695888/152819763-9dcfa276-b8c2-49c3-99fd-cd3835c64742.gif)

Reply:
`visibility: hidden` just hides them without removing icon's space

Reply:
$(".preview_mode_hidden").css("visibility", "hidden") works in that case.

Reply:
Ok

Reply:
`Error: You must create a stub for $("#compose .preview_mode_hidden").css`

giving test errors even after writing tests.